hazelcast-jca/src/test/java/com/hazelcast/jca/HazelcastConnectionImplTest.java

```diff
@@ -235,4 +243,59 @@ public void getXAResource() {
         XAResource resource = connection.getXAResource();
         assertNull(resource);
     }
+
+    @Test
+    public void getFlakeIdGenerator() {
+        String flakeIdGeneratorName = "flakeIdGenerator";
+        FlakeIdGenerator resource = connection.getFlakeIdGenerator(flakeIdGeneratorName);
+        assertSame(hz.getFlakeIdGenerator(flakeIdGeneratorName), resource);
+    }
```

Comment:
just for the coding conventions

- Previous tests did not name the variables as `resource`
- assertSame argument passing order is opposite when you check previous tests.

Reply:
👍